hw/syn/tools/dc/constraints.sdc

```diff
@@ -0,0 +1,160 @@
+# Copyright lowRISC contributors.
+# Licensed under the Apache License, Version 2.0, see LICENSE for details.
+# SPDX-License-Identifier: Apache-2.0
+#
+# Generic constraints file for simple testsynthesis flow
+# This held very simple for now and needs to be refined
+
+set DEFAULT_CLOCK_UNCERTAINTY 0.5
```

Comment:
I believe you can have a setup and hold uncertainty that are different.  You can use the setup uncertainty to give extra margin/guard band without blowing your hold times due to unreasonable skew.

Reply:
Indeed, you can specify them separately, but currently we do not fix hold times in this flow, hence it does not affect timing optimization. But I can provision that by naming it accordingly

Reply:
done